lightning/src/ln/channel.rs

```diff
@@ -6962,6 +6979,64 @@ mod tests {
 		assert_eq!(chan_8.holder_max_htlc_value_in_flight_msat, chan_8_value_msat);
 	}
 
+	#[test]
+	fn test_configured_holder_selected_channel_reserve_satoshis() {
+
+		// Test that `new_outbound` and `new_from_req` creates a channel with the correct value set
+		// for `their_channel_reserve_proportion`, when configured with a valid percentage value.
```

Comment:
I was trying to convey in my previous suggestion https://github.com/lightningdevkit/rust-lightning/pull/1619#discussion_r927005960 that I think this comment is a bit confusing, but I realize that I probably wasn't wasn't very clear :) :
The test asserts that the `channel_reserve`s are set to the correct value when the config value is configured with a valid value, and I think it would make more sense to word the comment to covey that instead of referring to that the config value has a correct value set. The test asserts the `channel_reserve`s values, and doesn't explicitly assert that: "`new_outbound` and `new_from_req` creates a channel with the correct value set for `their_channel_reserve_proportion`", although that is of course implicitly is correct of as well.

This isn't a big of a deal, but feel free to change this if you'd like. Either way I'm ok :).

Reply:
I am not sure if i understand this correctly, 
We shouldn't be testing the setters anyways(its trivial), we should test the functionality resulting from config.
I will clarify the comment.

Reply:
Thanks, your clarification addressed exactly what I found confusing :)!